### ISSUE TITLE
Alt text of the media on the self hosted sites is not visible in Media Browser

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#8393]
Steps to reproduce:

1. On self-hosted website navigate to media browser.
2. Open media settings of an image.
3. Add Alt text.

[![Image from Gyazo](https://i.gyazo.com/a736bc733a28691c13c05d29830bfc42.png)](https://gyazo.com/a736bc733a28691c13c05d29830bfc42)

 4. Open the media details of same image in the App.
 5. Notice that Alt text is missing.

[![Image from Gyazo](https://i.gyazo.com/6eebbb04b5e6d28724fdf0e10cc9f0b0.png)](https://gyazo.com/6eebbb04b5e6d28724fdf0e10cc9f0b0)

**Comment by khaykov** [comment]
So looks like the XML-RPC endpoint just doesn't return the alt property to us 🤷‍♂️ 

https://codex.wordpress.org/XML-RPC_WordPress_API/Media#wp.getMediaItem

**Comment by designsimply** [comment]
We received the following user report that I think is a match for this issue (or very close, at least in terms of media meta data as it relates to image titles disappearing): 

> disappearing image tags/alt text

and

> this happens when I'm logged in into app via url
> alt text keeps and titles disappear after I type them

* App version: 18.3
* Device model: Samsung SM-N975U
* OS version: Android 11
* Type of site: Jetpack

<sup>(internal references: 4372569-zen and p4a5px-2Mz-p2 /hat tip @thabotswana)</sup>

**Comment by jd-alexander** [comment]
@designsimply I think this issue will still persist because on a self hosted site's the XML-RPIC endpoint doesn't return the `alt` value. So this error is API related and cannot be resolved by us. 

> So looks like the XML-RPC endpoint just doesn't return the alt property to us 🤷‍♂️
> 
> https://codex.wordpress.org/XML-RPC_WordPress_API/Media#wp.getMediaItem

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by amberhinds** [comment]
Hey all - I would love to see this issue worked on. Here's a related Trac ticket. https://core.trac.wordpress.org/ticket/59414

Also, @kean can you please put the accessibility label back on the issue? As it is, there's no way to write alt text for featured images in the mobile app, which is an accessibility issue.

**Comment by amberhinds** [comment]
Also, this ticket is related and closed as fixed by @JoeDolson - perhaps it gives you what you need? https://core.trac.wordpress.org/ticket/58582

**Comment by jkmassel** [comment]
@amberhinds – we're adopting this, but in a slightly different manner than provided by the trac ticket. We're switching from using XML-RPC to the REST API, and this is among the first features we'll deliver that are powered by the REST API.

**Comment by amberhinds** [comment]
That's amazing! I'm glad to hear it. I'll be interested to see how the switch to use the REST API goes. Thank you!
